sphinx/builders/__init__.py

```diff
@@ -159,7 +164,7 @@ def get_target_uri(self, docname: str, typ: str | None = None) -> str:
     def get_relative_uri(self, from_: str, to: str, typ: str | None = None) -> str:
         """Return a relative URI between two source filenames.
 
-        May raise environment.NoUri if there's no way to return a sensible URI.
+        Raise :exc:`!sphinx.errors.NoUri` if there's no way to return a sensible URI.
```

Comment:
Use the docstring syntax

```suggestion
        :raises: :exc:`!sphinx.errors.NoUri` if there's no way to return a sensible URI.
```